_includes/components/pledge.njk

```diff
@@ -0,0 +1,12 @@
+<div class="pledge sg-component">
+    <div class="pledge-quote">
+      <p>GETconf 2019 was a success, but our work is not done.  We want to exchange ideas, build community, and take action to advance gender  equity in the tech workforce in Omaha.</p>
```

Comment:
Is there an extra space between gender & equity here?

Reply:
Nice catch!